docs/modules/plugins/pages/plugin-web-app-playwright.adoc

```diff
@@ -709,3 +709,46 @@ When I execute sequence of actions:
 include::partial$common-web-app-browser-steps.adoc[]
 
 include::partial$common-ui-screenshot-steps.adoc[]
+
+=== Mobile Emulation
```

Comment:
Now (I guess) this doc is equal to the same doc from `vividus-plugin-web-app`, thus the same content should be moved to partials and then re-used in both places